package.json

```diff
@@ -63,5 +63,8 @@
     "hooks": {
       "pre-commit": "npm-run-all -ln --parallel lint test"
     }
+  },
+  "dependencies": {
+    "recharts": "^2.7.3"
```

Comment:
Isn't it supposed to be a dependency for packages/jaeger-ui?

Reply:
Ahh yes. Silly mistake.
Thank you.